### DISCUSSION TITLE
Gramine RA-TLS golang example

### DISCUSSION THREAD

**Discussion by lonerapier** [gramineproject/gramine#1288] [General]
We've created a [golang](https://github.com/konvera/mutual-ratls-example) implementation of [RA-TLS CI-example](https://github.com/gramineproject/gramine/tree/master/CI-Examples/ra-tls-mbedtls). It includes a server/client architecture similar to the example where client and server verifies each other's quote certificates.

Would you be interested in adding this example to upstream Gramine repository? Should there be any general additions for this example to be valid enough?

**Comment by dimakuv** [comment]
I think this would be a better fit for the Contrib repo: https://github.com/gramineproject/contrib

But please take into account that reviewers are currently overloaded with other, high priority PRs, so it will take some time for your PR to be approved and merged.
